src/Core/Common/DuplicateAnImage/Code.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+# Copyright NumFOCUS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import itk
+
+Dimension = 2
+PixelType = itk.UC
+
+ImageType = itk.Image[PixelType, Dimension]
+
+randomImageSource = itk.RandomImageSource[ImageType]()
+randomImageSource.SetNumberOfWorkUnits(1)  # to produce non-random results
+
+image = randomImageSource.GetOutput()
+
+duplicator = itk.ImageDuplicator[ImageType]()
+duplicator.SetInputImage(image)
+duplicator.Update()
+
+clonedImage = duplicator.GetOutput()
```

Comment:
Here, in would be nice to use the more Pythonic:

```
cloned = itk.image_duplicator(image)
```

Reply:
Done in f7c20d4.